media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="A commandline client for Music Player Daemon"
+DESCRIPTION="Music Player Daemon (MPD) is a flexible, powerful, server-side \
+application for playing music. Through plugins and libraries it can play a \
+variety of sound files while being controlled by its network protocol.
+
+NCMPCPP is an ncurses based commandline client for Music Player Daemon."
+HOMEPAGE="https://rybczak.net/ncmpcpp/"
+COPYRIGHT="2008-2016 The NCMPCPP Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/arybczak/ncmpcpp/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="0d726c886b3e0b63d5f126e3b21ab67a9234e3046c42c591622059659befad67"
+SOURCE_DIR="ncmpcpp-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	ncmpcpp$secondaryArchSuffix = $portVersion
+	cmd:ncmpcpp
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	boost$secondaryArchSuffix
+	ncurses6$secondaryArchSuffix
```

Comment:
boost$secondaryArchSuffix and ncurses6$secondaryArchSuffix should be removed or replaced by something else.